RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.11.2
+
+- fixed: `git machete` now correctly infers remote for fetching of branch
```

Comment:
I'd specify the exact case here, but ofc with the low level of detail, as adequate for release notes